Simplenote/Classes/SignupVerificationViewController.swift

```diff
@@ -28,7 +28,8 @@ class SignupVerificationViewController: UIViewController {
     @IBOutlet private weak var footerTextView: UITextView! {
         didSet {
             let email = SPCredentials.simplenoteFeedbackMail
-            guard let emailURL = URL(string: "mailto:\(SPCredentials.simplenoteFeedbackMail)") else {
+
+            guard let emailURL = URL(string: "mailto:\(email)") else {
```

Comment:
I kept the `let email` definition because it's used [below](https://github.com/Automattic/simplenote-ios/pull/1710/files#diff-a3960ff76683569e8174546ec54c08796d7d23e60519ad127520c18572b99ab8R43), too.

This is now the only usage of the feedback email, FWIW.